### PR TITLE
Update Meta Tags

### LINKED ISSUE #1546: SEO: update meta titles and descriptions based on keyword targeting concept
**Problem**
Many meta titles are out-of-date, were randomly created, too short or/and don't include our keywords.

**Solution**
We created a [keyword targeting concept](https://www.notion.so/gitpod/Keyword-targeting-concept-d985ef8047864b4e829329a60a85c7c1) to identify the most relevant keywords based on search intent and Gitpod's benefit.

**Next steps**
- [x] update the meta titles of the most relevant pages
- [x] update the meta description
- [x] minor additional changes to some websites to fit the keyword targeting
- [ ] make sure /community is indexed. currently, it highlights community.gitpod.io instead
- [ ] add a meaningful meta description to gitpod.io/workspaces

Note: all required changes are marked in blue in the table [here](https://www.notion.so/gitpod/Keyword-targeting-concept-d985ef8047864b4e829329a60a85c7c1)

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #1546
closes partially #1464 => deindexatino of `/changelog/*`

- [x] /docs/IDE's and Editors

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1604"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1604"></a>

### CODE REVIEW

File: src/routes/changelog/__layout.svelte [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
```
Comment (Cahllagerfeld) [inline src/routes/changelog/__layout.svelte:1]: Added layout for `/changelog/*`

File: gitpod/docs/getting-started.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ title: Getting Started
...
 </script>
 
+<script lang="ts">
+  import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+data={{
+    description:
+      "Learn how to start your first workspace, set up a  Gitpod.yml configuration file and enable prebuilds. Get started for free.",
```
Comment (mikenikles) [inline gitpod/docs/getting-started.md:17]:
```suggestion
      "Learn how to start your first workspace, set up a gitpod.yml configuration file and enable Prebuilds. Get started for free.",
```

File: gitpod/docs/ides-and-editors/index.md [hunk 3]
```diff
@@ -7,6 +7,19 @@ title: IDE's & Editors
...
 
+<script lang="ts">
+    import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+data={{
+    description:
+      "Connect and integrate Gitpod workspaces with your favourite IDE or editor. VS Code Browser and Desktop. JetBrains. IntelliJ. GoLand. PhpStorm. PyCharm. Vim. Emacs. SSH.",
+    title: "Gitpod integrates with  VS Code, JetBrains and SSH.",
```
Comment (mikenikles) [inline gitpod/docs/ides-and-editors/index.md:18]:
```suggestion
    title: "Gitpod integrates with VS Code, JetBrains and SSH.",
```

File: gitpod/docs/index.md [hunk 4]
```diff
@@ -14,12 +14,22 @@ title: Introduction
...
 
   export let changelogEntries;
 
   setContext(contextKeyChangelogEntries, changelogEntries);
 </script>
 
+<OpenGraph
+data={{
+    description:
+      "Explore our docs to learn how to set up and configure your cloud developer environment. Quickstart. Getting started. Configure. Develop. IDEs. Editors. Integrations. Self-hosted. Supply Chain Security.",
```
Comment (mikenikles) [inline gitpod/docs/index.md:27]:
```suggestion
      "Explore our docs to learn how to set up and configure your cloud developer environment. Quickstart. Getting started. Configure. Develop. IDEs. Editors. Integrations. Self-Hosted. Supply Chain Security.",
```

File: gitpod/docs/self-hosted/latest/index.md [hunk 5]
```diff
@@ -6,6 +6,18 @@ title: Gitpod Self-Hosted
...
   export const prerender = true;
 </script>
+<script lang="ts">
+  import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+data={{
+    description:
+      "Install and run Gitpod with full control on public cloud providers or self-managed Kubernetes clusters. Enterprise grade security within corporate firewalls and air-gapped networks. Requirements. Installation. Configuration. GKE. EKS. AKS. Open-source.",
```
Comment (mikenikles) [inline gitpod/docs/self-hosted/latest/index.md:16]:
```suggestion
      "Install and run Gitpod with full control on public cloud providers or self-managed Kubernetes clusters. Enterprise-grade security within corporate firewalls and air-gapped networks. Requirements. Installation. Configuration. GKE. EKS. AKS. Open-source.",
```

File: gitpod/docs/self-hosted/latest/index.md [hunk 6]
```diff
@@ -6,6 +6,18 @@ title: Gitpod Self-Hosted
...
+    title: "Gitpod Self-hosted installation guide",
```
Comment (mikenikles) [inline gitpod/docs/self-hosted/latest/index.md:17]:
```suggestion
    title: "Gitpod Self-Hosted installation guide",
```

File: src/routes/changelog/__layout.svelte [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  import { page } from "$app/stores";
+</script>
+
+<svelte:head>
+  {#if !$page.url.pathname.endsWith("changelog")}
```
Comment (mikenikles) [inline src/routes/changelog/__layout.svelte:6]: Elegant solution 💡

File: src/routes/self-hosted.svelte [hunk 8]
```diff
@@ -13,8 +13,11 @@
 
 <OpenGraph
   data={{
-    description: "Self-Host Gitpod on Your Own Infrastructure.",
-    title: "Self-Hosted",
+    description:
+      "Deploy and operate Gitpod yourself. Install Gitpod Self-hosted on GKE, EKS, AKS or Kubernetes. Made for teams. Free for up to 10 users.",
```
Comment (mikenikles) [inline src/routes/self-hosted.svelte:17]:
```suggestion
      "Deploy and operate Gitpod yourself. Install Gitpod Self-Hosted on GKE, EKS, AKS or Kubernetes. Made for teams. Free for up to 10 users.",
```

File: src/routes/self-hosted.svelte [hunk 9]
```diff
@@ -13,8 +13,11 @@
...
+    title: "Gitpod Self-hosted - Run Gitpod on your infrastructure ",
```
Comment (mikenikles) [inline src/routes/self-hosted.svelte:18]:
```suggestion
    title: "Gitpod Self-Hosted - Run Gitpod on your infrastructure",
```

File: src/routes/vs/jetbrains-space.svelte [hunk 10]
```diff
@@ -14,8 +14,9 @@
 <OpenGraph
   data={{
     description:
-      "Gitpod is the vendor neutral, open-source platform that integrates with your individual stack including all JetBrains IDEs.",
-    title: "Gitpod vs JetBrains Space",
+      "Gitpod is the vendor neutral, open-source alternative that integrates with VS Code as well as all JetBrains IDEs like IntelliJ, GoLand, PhpStorm, PyCharm.",
```
Comment (mikenikles) [inline src/routes/vs/jetbrains-space.svelte:17]:
```suggestion
      "Gitpod is the vendor-neutral, open-source alternative that integrates with VS Code as well as all JetBrains IDEs like IntelliJ, GoLand, PhpStorm, PyCharm.",
```

File: src/routes/vs/local-development.svelte [hunk 11]
```diff
@@ -29,9 +29,10 @@
 
 <OpenGraph
   data={{
-    title: "Gitpod vs local development",
+    title: "Gitpod vs local development - move to the cloud",
     description:
-      "Why should you move your dev environment to the cloud? Here is the answer.",
+      "Develop like Google, Facebook and Shopify.  Spend less time worrying about brittle, local dev environments and leverage the power of the cloud to improve your team’s developer experience.",
```
Comment (mikenikles) [inline src/routes/vs/local-development.svelte:34]:
```suggestion
      "Develop like Google, Facebook and Shopify. Spend less time worrying about brittle, local dev environments and leverage the power of the cloud to improve your team’s developer experience.",
```

**Review by mikenikles — APPROVED** [review]

**Pull request merged**